### PR TITLE
feat(remix,remix-dev): warn if `remix setup` is called or if importing from `remix`

### PR DESCRIPTION
**Pull request by pcattori**
Screen Shot 2022-04-05 at 3 14 02 PM

Screen Shot 2022-04-05 at 3 15 52 PM

### CODE REVIEW

**Review by MichaelDeBoey — APPROVED** [review]
LGTM! 👍

**Comment by kentcdodds** [comment]
Should the message say something about running remix migrate?

File: packages/remix/index.ts [hunk 1]
```diff
@@ -1,6 +1,9 @@
 // This class exists to prevent https://github.com/remix-run/remix/issues/2031 from occuring
 export class RemixNotSetupError extends Error {
   constructor() {
+    console.warn(
+      "DEPRECATION: Importing from `remix` is deprecated. Import from `@remix-run/*` packages instead."
```
Comment (pcattori) [inline packages/remix/index.ts:5]: This occurrence is outside of `remix-dev` package, so can't use our nice `deprecation(...)` utility here.

**Comment by pcattori** [comment]
@kentcdodds I will add a `HINT:` about `migrate` command after #2653 is in.

**Review by MichaelDeBoey — APPROVED** [review]

**Review by kentcdodds — APPROVED** [review]
Super 👍

**Pull request merged**